apps/falcon_app.py

```diff
@@ -1,10 +1,12 @@
 from binascii import hexlify
 from hashlib import md5
 import os
-
+import threading
+import sys
 import falcon
 import vulnpy.falcon
-
+import time
+import contrast
```

Comment:
these apps are intended to be run with or without contrast, so I don't think we should have it imported at the top level